plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/longrunning/LongRunningEventListener.java

```diff
@@ -0,0 +1,60 @@
+package io.trino.plugin.jdbc.longrunning;
```

Comment:
I don't think these changes are any how related to the base-jdbc connector framework. Please move these testing to `trino-main`. Please create dedicated query runner for your case and dedicated test class. You may consider using mock connector. You are going to test a thing that not related to any specific connector. Good example of such test is `TestAccessControl` which tests access control. Maybe we are missing `TestEventListener`... 🤔